web/app/js/components/ResourceDetail.jsx

```diff
@@ -188,10 +189,18 @@ export class ResourceDetailBase extends React.Component {
     this.api.setCurrentRequests(apiRequests);
 
     Promise.all(this.api.getCurrentPromises())
-      .then(results => {
-        const [resourceRsp, podListRsp, podMetricsRsp, upstreamRsp, downstreamRsp, ...rsp] = [...results];
+      .then(apiResponses => {
```

Comment:
Was there a reason you changed `results` to `apiResponses` here and below?

Reply:
Yes, for consistency reasons with `apiRequests` and to make clearer its meaning as results was a bit too generic. But I'm happy to change it back if you prefer `results`.